test_data/config.toml

```diff
@@ -775,3 +775,21 @@ myhello.command = """
 myhello.runtime-packages = [ "hello", "goodbye" ]
 ' | flox lock-manifest - > "$RESPONSE_FILE"
 '''
+
+[build.hello]
+files = ["build/hello/hello.c", "build/hello/.flox"]
+pre_cmd = '''
+    rm .flox/env/manifest.lock
+'''
+cmd = '''
+    # re-resolve and build
+    flox lock-manifest .flox/env/manifest.toml > .flox/env/manifest.lock
+'''
+post_cmd = '''
+    build_output_dir="$(dirname $RESPONSE_FILE)"
+    build_dir="$build_output_dir/hello"
+    mkdir -p "$build_dir"
+    cp -R .flox "$build_dir/.flox"
+    cp hello.c "$build_dir/hello.c"
+    mv "$RESPONSE_FILE" "$build_dir/hello.yaml"
+'''
```

Comment:
question: why do we need to copy over the input files at all, we've been fine sourcing outputs and inputs separately using `GENERATED_DATA` and `INPUT_DATA` so far, why break that precendent over either explicitly use `INPUT_DATA` where needed, or explicitly bundling inputs via the `mk_data` tool?

Reply:
As I understand it's so that `flox` commands can be iterated on against the `GENERATED_DATA` from outside of the test suite.

Reply:
Yes, I also think it's generally an anti-pattern to depend on things in the input data folder. One more thing is that the recorded responses only show up in the output directory.